### PR TITLE
Move DORA session. Add location and parking info.

### PR DESCRIPTION
**Pull request by erkist**
### Applicable Issues
N/A

### Description of the Change
Switch DORA session and Conformance session in agenda. Add information about Tuesday and Wednesday locations. Add link to parking map.

### Alternate Designs
-

### Benefits
-

### Possible Drawbacks
-

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Erik Sternerson, erik@sternerson.se

### CODE REVIEW

File: summit.html [hunk 1]
```diff
@@ -33,21 +33,27 @@ <h1 class="section-heading">The practicalities</h1>
           <ul>
             <li>
               Where: <a href="https://dowhile.se/">doWhile</a>,
-              <a href="https://goo.gl/maps/ixnYGRBcCbQQw3Gd6">
+              <br /> Tuesday: <a href="https://www.google.com/maps/place/Lindholmen+Conference+Centre+at+Lindholmen+Science+Park/@57.7073754,11.9223816,14z/data=!4m6!3m5!1s0x464ff35aa1bf62a7:0xcb26569fedefe772!8m2!3d57.7071432!4d11.9400314!16s%2Fg%2F1pzq5rfgk?entry=ttu">
```
Comment (e-backmark-ericsson) [inline summit.html:36]: Den url:en kan förenklas till "https://goo.gl/maps/1otFDxh2fGPudF7x8"

**Review by e-backmark-ericsson — COMMENTED** [review]
You forgot to change your surname, Erik :)

File: summit.html (hunk 1 above)
Comment (erkist) [reply summit.html]: Done!

**Comment by erkist** [comment]
> You forgot to change your surname, Erik :)

Done as well!

**Review by e-backmark-ericsson — APPROVED** [review]

**Review by m-linner-ericsson — APPROVED** [review]

**Pull request merged**